CONTRIBUTING.md

```diff
@@ -0,0 +1,157 @@
+# Contributing to document-service
```

Comment:
We follow a specific convention for headings.

Only the first word starts with CAPS unless the other word is a noun.
Refer the README.md file in this project to understand how the headings conventions is followed.

Reply:
Updated